src/style.css

```diff
@@ -0,0 +1,113 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.block {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  height: 403px;
+  width: 200px;
+
+}
+
+.image {
+  height: 134px;
+  width: 160px;
+  margin: 30px 21px 0 19px;
+  line-height: 134px;
+}
+
+.name {
+  font-size: 12px;
+  line-height: 15px;
+  margin: 41px 16px 0 14px;
+  width: 170px;
+  height: 32px;
+  font-weight: 500;
+}
+
+.code {
+  font-size: 11px;
+  line-height: 13px;
+  margin: 3px 85px 0 14px;
+  width: 101px;
+  height: 12px;
+}
+
+.stars {
+  margin: 18px 0 0 15px;
+  display: inline-block;
+  width: 91px;
+  height: 15px;
+  line-height: 15px;
+}
+
+.star-active {
+  width: 13px;
+  height: 13px;
+  margin-right: 2px;
+}
+
+.star {
+  width: 13px;
+  height: 13px;
+}
+
+.fee {
+  font-size: 11px;
+  line-height: 13px;
+  width: 58px;
+  height: 13px;
+  color: #000;
+  margin: -32px 23px 0 112px;
+  position: absolute;
+}
+
+.prise {
+  font-size: 11px;
+  line-height: 13px;
+  margin: 21px 0 0 14px;
+  color: #000;
+  display: block;
+  height: 12px;
+}
+
+.real {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+  text-align: right;
+  color: #000;
+  margin: -17px 14px 0 0;
+  height: 19px;
+}
+
+.buy {
+  background: #00acdc;
```

Comment:
add `box-sizing: border-box`, and then you can remove `width, height` from hover